facebook/__init__.py

```diff
@@ -249,6 +249,7 @@ def request(
             raise GraphAPIError(response)
 
         headers = response.headers
+        result = {}
```

Comment:
No it is not necessary.
That was added in order to have `result`  declaration/scope in the function body.